src/Evolution/Systems/Burgers/FiniteDifference/BoundaryConditionGhostData.hpp

```diff
@@ -99,6 +103,10 @@ void BoundaryConditionGhostData::apply(
              "Boundary condition is not set (the pointer is null) in block "
                  << element.id().block_id() << ", direction: " << direction
                  << " (Burgers::fd::BoundaryConditionGhostData)");
+
```

Comment:
@nilsdeppe I pushed another fixup for testing the second ASSERT here. I couldn't think of a good way to test the first one :(